src/lib/lint.ts

```diff
@@ -42,9 +44,61 @@ export function parseOutdatedSchoolPeriods(desc: string): string[] {
     .filter((s) => !s.includes(THIS_YEAR.toString()));
 }
 
+/** Parses the given description for ethnicities and returns matches. */
+export function parseGradeLevels(desc: string): GradeLevel[] {
+  const lowerDesc = desc.toLowerCase();
+  const keywords = {
+    [GradeLevel.MiddleSchool]: ['middle school', '[78]th grade'],
+    [GradeLevel.HsFreshman]: ['9th grade'],
+    [GradeLevel.HsSophomore]: ['10th grade'],
+    [GradeLevel.HsJunior]: ['11th grade', 'high school[^.]* junior'],
+    [GradeLevel.HsSenior]: ['12th grade', 'high school[^.]* senior'],
+    [GradeLevel.CollegeFreshman]: ['freshm[ae]n'],
+    [GradeLevel.CollegeSophomore]: ['sophomore'],
```

Comment:
wouldn't it be best to specifiy `college freshm[ae]n / college sophomore`?

Reply:
See the reasoning in my previous comment. We can revisit this later if it's assuming college freshmen/sophomores when it shouldn't be.